tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -104,6 +104,33 @@ Feature: Module
     When I uninstall module "ps_featuredproducts" with deleteFile false
     Then I should have an exception that module is not installed
 
+  Scenario: Install module
+    When I install module "ps_featuredproducts"
+    Then module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts  |
+      | version        | 1.0.0                |
+      | enabled        | true                 |
+      | installed      | true                 |
+
+  Scenario: Upload module with zip file on disk
+    When I upload module "test_install_cqrs_command" from "zip" "test_install_cqrs_command.zip"
```

Comment:
I guess you are using a module that doesn't exist yet right? If so please check that the module is not present before uploading the file You can do so by using the "module has following infos" step and assert that it triggers a ModuleNotFoundException

Once this is asserted you can upload the zip, by the way since I guess you created the zip manually can you make it so it doesn't have a `1.0.0` version, just to make sure it reads the new version correctly from the extracted archive and not as a default value

Then before the module is installed you should check that it is now present (you don't get a ModuleNotFoundException) but is not installed